CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Features
 1. [#264](https://github.com/influxdata/influxdb-client-python/pull/264): Org parameter can be specified as ID, Name or Organization Object [write, query]
+1. [#281](https://github.com/influxdata/influxdb-client-python/pull/281): `FluxTable`, `FluxColumn` and `FluxRecord` objects have helpful reprs
```

Comment:
Please, rebase your sources and move this note into release `1.20.0`.

Reply:
:heavy_check_mark: